src/bidiMapper/domains/script/channelProxy.ts

```diff
@@ -31,8 +32,9 @@ export class ChannelProxy {
   readonly #properties: Script.ChannelProperties;
 
   readonly #id = uuidv4();
+  readonly #logger?: LoggerFn;
 
-  constructor(channel: Script.ChannelProperties) {
+  constructor(channel: Script.ChannelProperties, logger?: LoggerFn) {
     if (
       ![0, null, undefined].includes(channel.serializationOptions?.maxDomDepth)
```

Comment:
@thiagowfx Side question is this still needed? I thought the deep serialization options landed.